web/public/locales/en.json

```diff
@@ -146,7 +147,10 @@
     "selectLanguage": "Select language",
     "lowCarbDescription": "Includes renewables and nuclear",
     "dataIsDelayed": "Data is delayed",
-    "price": "Price"
+    "price": "Price",
+    "netExchange": "Net exchange",
+    "importing": "Importing",
+    "exporting": "Exporting"
```

Comment:
Could probably use more descriptive strings here to avoid confusion. Any suggestion?

Reply:
I think this is okay :)